lib/libutils/isoc/include/sys/_default_fcntl.h

```diff
@@ -0,0 +1,77 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
BSD-3-Clause seems more appropriate, as regards to the other **newlib** materials already present in optee_os source tree.

Ditto for _fcntl.h_.

Reply:
OK

Reply:
Apply into https://github.com/OP-TEE/optee_os/pull/6700/commits/d2b49afed89a11fe3d257e08dbb41e879592d443